tests/NewCypress/cypress/e2e/regression/footer.cy.ts

```diff
@@ -0,0 +1,38 @@
+import footerLinks from "../../pages/footerLinks";
+import paginationPage from "../../pages/paginationPage";
+
+describe("Testing the components of the home page", () => {
+
+    beforeEach(() => {
+        cy.login();
+    });
+
+    it("Should check that the home page footer bar privacy link is present and functional", () => {
+        footerLinks
+            .checkPrivacyLinkPresent()
+            .clickPrivacyLink()
+
+        paginationPage
+            .checkImAtTheCorrectUrl('privacy')
```

Comment:
Could the naming be a bit more verbose? checkImAtTheCorrectUrl, what is Im here, pedeantic I know but CheckCurrentUrlIsCorrect or something akin may give more clarity.